lib/utils.js

```diff
@@ -1,25 +1,102 @@
 'use strict';
 
 var qs = require('qs');
-var _ = require('lodash');
 
 var hasOwn = {}.hasOwnProperty;
 var toString = {}.toString;
 
 var utils = module.exports = {
 
+  // Note that in ES6 all uses of this can just be replaced with Object.assign.
+  // Given a few more years we should drop this function completely.
+  assign: function(o1, o2) {
+    for (var attr in o2) {
+      o1[attr] = o2[attr];
+    }
+    return o1;
+  },
+
   isAuthKey: function(key) {
     return typeof key == 'string' && /^(?:[a-z]{2}_)?[A-z0-9]{32}$/.test(key);
   },
 
   isOptionsHash: function(o) {
-    return _.isPlainObject(o) && ['api_key', 'idempotency_key', 'stripe_account'].some(function(key) {
+    return utils.isPlainObject(o) && ['api_key', 'idempotency_key', 'stripe_account'].some(function(key) {
       return o.hasOwnProperty(key);
     });
   },
 
-  isObject: function(o) {
-    return _.isPlainObject(o);
+  /**
+   * Checks if `value` is a plain object, that is, an object created by the
+   * `Object` constructor or one with a `[[Prototype]]` of `null`.
+   *
+   * Note that this function is vendored from Lodash with some minor
+   * modifications.
```

Comment:
Seem again like we could just pull in https://www.npmjs.com/package/lodash.isplainobject and be safer.